src/test/java/AppHealthCheckTest.java

```diff
@@ -0,0 +1,23 @@
+import org.junit.jupiter.api.Test;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class AppHealthCheckTest {
+
+    @Test
+    public void isAppAlive() throws Exception {
```

Comment:
I guess the name have to begin with 'should'. Thats much better convention for this kind of test